pacehrh/vignettes/pacehrh.rmd

```diff
@@ -15,15 +15,81 @@ knitr::opts_chunk$set(echo = TRUE)
 
 # Get started with the PACE-HRH model 
 
+The PACE-HRH model simulates healthcare needs in a given population to enable
+better health system staffing decisions. PACE-HRH is a stochastic, Monte
+Carlo simulation model, meaning that the population, its healthcare needs,
+and the time required to meet those are simulated and there is randomness
+built into the model uncertainty in the population's size, growth, and
+healthcare needs. The model is configured via Excel sheets and run via an R
+script. 
+
+## Install the model
+
+Generally, we recommend installing the current released version of the Windows
+binary, which has been tested and documented. You can also install directly
+from unreleased source code in GitHub, which may be useful if you are
+modifying the model code or need the latest fixes that have not been released
+officially. 
+
+<!--- Can this only be run on Windows? Do we have alternate directions for Mac or Linux OS?--->
+
+### Install using the current released version Windows binaries 
+
+Follow the instructions below to install a tested, released version of the model.
+
+1.  Download the latest binary zip file from https://github.com/InstituteforDiseaseModeling/PACE-HRH/releases.
+    This will be the first file listed under **Assets**. 
+    ![Binary zip file under GitHub release assets.](images/pacehrh-release.png)
```

Comment:
Was this image checked in? I did not see it

Reply:
Oops! Good catch--I just added it.